app/views/idv/welcome/show.html.erb

```diff
@@ -25,8 +25,14 @@
   <h2><%= t('doc_auth.instructions.getting_started') %></h2>
 
   <%= render ProcessListComponent.new(heading_level: :h3, class: 'margin-y-3') do |c| %>
-    <%= c.with_item(heading: t('doc_auth.instructions.bullet1')) do %>
-      <p><%= t('doc_auth.instructions.text1') %></p>
+    <% if decorated_sp_session.selfie_required? %>
```

Comment:
Note: `selfie_required?` not only checks that the SP requested a selfie, but also checks that the feature flag is enabled.